core/templates/dev/head/services/searchService.js

```diff
@@ -102,6 +102,8 @@ oppia.factory('searchService', [
       if (successCallback) {
         successCallback();
       }
+      return '\?q=' + encodeURI(searchQuery +
```

Comment:
The backslash is needed so that when it is passed in the browser, it treats the ? as a question mark rather than something else (without it. it sanitizes the link into something that the server does not understand).